raiden/tests/integration/transfer/test_mediatedtransfer.py

```diff
@@ -273,3 +275,39 @@ def test_mediated_transfer_messages_out_of_order(
             app1, deposit - amount, [],
             app2, deposit + amount, [],
         )
+
+
+@pytest.mark.parametrize('number_of_nodes', (1,))
+@pytest.mark.parametrize('channels_per_node', (CHAIN,))
+def test_mediated_transfer_calls_pfs(raiden_network, token_addresses):
+
+    app0, = raiden_network
```

Comment:
Could you please add a mediating node and make sure that the mediator_init call also calls PFS routing?

Reply:
I created a transfer using factories instead and mediated it through the one node. The test will be faster that way.